main.tf

```diff
@@ -12,9 +12,9 @@ resource "google_org_policy_policy" "policy" {
         for_each = try(spec.value.rules, [])
 
         content {
-          allow_all = try(rules.value.allow_all, null)
-          deny_all  = try(rules.value.deny_all, null)
-          enforce   = try(rules.value.enforce, null)
+          allow_all = try(rules.value.allow_all, null) != null ? rules.value.allow_all == true ? "TRUE" : null : null
```

Comment:
if `*_all` and `enforce` are booleans, this could be:
```
allow_all = try(rules.value.allow_all, null) != null ? rules.value.allow_all ? "TRUE" : "FALSE" : null
```

Reply:
hm this is what I tried (more or less) in https://github.com/mineiros-io/terraform-google-organization-policy/actions/runs/3401979187/jobs/5657434342 which caused the tests to fail